sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/util/IntervalUtilsSuite.scala

```diff
@@ -186,4 +186,18 @@ class IntervalUtilsSuite extends SparkFunSuite {
     assert(!isNegative("1 year -360 days", 31))
     assert(!isNegative("-1 year 380 days", 31))
   }
+
+  test("string to interval") {
```

Comment:
can we always test `stringToInterval` and `fromString` to make sure they have the same behavior?